multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/a3manga/A3MangaGenerator.kt

```diff
@@ -15,7 +15,6 @@ class A3MangaGenerator : ThemeSourceGenerator {
         SingleLang("A3 Manga", "https://www.a3manga.online", "vi", overrideVersionCode = 1),
         SingleLang("Team Lanh Lung", "https://teamlanhlung.me", "vi", sourceName = "Team Lạnh Lùng", overrideVersionCode = 1),
         SingleLang("Ngon Phong", "https://www.ngonphong.com", "vi", sourceName = "Ngôn Phong", overrideVersionCode = 1),
-        SingleLang("O Cu Meo", "https://www.ocumoe.com", "vi", sourceName = "Ổ Cú Mèo", overrideVersionCode = 1),
```

Comment:
Pretty sure Ổ Cú Mèo isn't dead, just a domain change: https://www.ocumeo.store/

Reply:
My bad, will update its domain instead

Reply:
Actually, the old domain (https://ocumoe.com/) does redirecting to https://www.a3manga.store.
Maybe they setup a new domain and redirect the old one.